vm/src/stdlib/os.rs

```diff
@@ -2436,6 +2437,52 @@ mod posix {
         Ok(())
     }
 
+    #[pyattr]
+    #[pyclass(name = "sched_param")]
+    #[derive(Debug)]
+    struct SchedParam {
+        sched_priority: PyObjectRef,
+    }
+
+    impl PyValue for SchedParam {
+        fn class(_vm: &VirtualMachine) -> &PyTypeRef {
+            Self::static_type()
+        }
+    }
+
+    impl TryFromObject for SchedParam {
+        fn try_from_object(_vm: &VirtualMachine, obj: PyObjectRef) -> PyResult<Self> {
+            Ok(SchedParam {
+                sched_priority: obj,
+            })
+        }
+    }
+
+    #[pyimpl(with(SlotConstructor))]
+    impl SchedParam {
+        #[pyproperty]
+        fn sched_priority(&self, vm: &VirtualMachine) -> PyObjectRef {
+            self.sched_priority.clone().into_pyobject(vm)
+        }
+
+        #[pymethod(magic)]
+        fn repr(&self, vm: &VirtualMachine) -> PyResult<String> {
+            let sched_priority_repr = vm.to_repr(&self.sched_priority)?;
+            Ok(format!(
+                "posix.sched_param(sched_priority = {})",
+                sched_priority_repr.as_str()
+            ))
+        }
+    }
+
+    impl SlotConstructor for SchedParam {
+        type Args = PyObjectRef;
```

Comment:
Because `SchedParam` implements `TryFromObject`,
I think this conversion also works
```suggestion
        type Args = SchedParam;
```